app/views/annotations/_annotation_list.html.erb

```diff
@@ -1,9 +1,17 @@
 <ul class='annotation_text_list'>
-  <% if annotation_category.annotation_texts.empty? %>
+  <%
+    text = []
+    annotation_category.annotation_texts.each do |annotation_text|
+      if (annotation_text.user.nil? || (annotation_text.user.admin? || annotation_text.user == @current_user))
```

Comment:
This is interesting for a few reasons. 

1. In general, when building a new list like this you should be using the Enumerable `select` method (I think that's what Ruby calls it)
2. I'm not sure under which conditions `user` is nil (that's strange). No need for parentheses around the whole condition.